contracts/extensions/BatchTradeExtension.sol

```diff
@@ -102,8 +96,6 @@ contract BatchTradeExtension is BaseGlobalExtension {
      * @param _delegatedManager     Instance of the DelegatedManager to initialize the TradeModule for
      */
     function initializeModule(IDelegatedManager _delegatedManager) external onlyOwnerAndValidManager(_delegatedManager) {
-        require(_delegatedManager.isInitializedExtension(address(this)), "Extension must be initialized");
```

Comment:
Why are we removing these?

Reply:
When we `_initializeModule` we `_invokeManager` and call `DelegatedManager.interactManager` which is `onlyExtension` privileged https://github.com/SetProtocol/set-v2-strategies/blob/29db4a3de4a6ecd55a2c86d85a002b7e1106f8b2/contracts/manager/DelegatedManager.sol#L119